pkg/kubectl/cmd/portforward/portforward.go

```diff
@@ -113,6 +120,7 @@ func NewCmdPortForward(f cmdutil.Factory, streams genericclioptions.IOStreams) *
 		},
 	}
 	cmdutil.AddPodRunningTimeoutFlag(cmd, defaultPodPortForwardWaitTimeout)
+	cmd.Flags().StringSliceVar(&opts.Address, "address", []string{"localhost"}, "Addresses to listen on (comma separated)")
```

Comment:
I suggest "local-addresses"?

Reply:
`address` was selected as it's the same as for ie. `kubectl proxy`, so makes sense to stay coherent